src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -2529,23 +2529,81 @@ public static ArgumentSyntax Argument(ExpressionSyntax expression)
         /// <summary>Creates a new ArgumentSyntax instance.</summary>
         public static ArgumentSyntax Argument(NameColonSyntax nameColon, SyntaxToken outKeyword, VariableDeclarationSyntax declaration)
         {
-            if (outKeyword.Kind() != SyntaxKind.OutKeyword)
+            return Argument(nameColon, outKeyword, (CSharpSyntaxNode)declaration);
```

Comment:
Nevermind the first one as well. I was looking at the wrong overload too. The one that is now internal is in the generated syntax code. Thanks